content/blog/stacks-0-1-0/index.Rmd

```diff
@@ -0,0 +1,61 @@
+---
+output: hugodown::hugo_document
+
+slug: stacks-0-1-0
+title: stacks 0.1.0
+date: 2020-11-17
```

Comment:
Go ahead and change the date (anything this week is fine) and re-knit

EDIT: Actually, wait until the binaries are built.